Assets/MixedRealityToolkit/Providers/UnityInput/MouseDeviceManager.cs

```diff
@@ -27,17 +27,62 @@ public class MouseDeviceManager : BaseInputDeviceManager, IMixedRealityMouseDevi
             IMixedRealityInputSystem inputSystem,
             string name = null,
             uint priority = DefaultPriority,
-            BaseMixedRealityProfile profile = null) : base(registrar, inputSystem, name, priority, profile) { }
+            BaseMixedRealityProfile profile = null) : base(registrar, inputSystem, name, priority, profile)
+        { }
+
+        // Values defining the range of the cursor and wheel speed multipliers
+        private const float MinSpeedMultiplier = 0.1f;
```

Comment:
do we need these min / max values here? you already define a range in the profile for users setting it up in the editor and if someone sets this to a ridiculous value in code I would assume they're doing that intentional and we don't need to clamp it to some reasonable range?

Reply:
Those are there to eliminate "magic numbers" and i prefer there be no difference in range between the Unity inspector and code :)